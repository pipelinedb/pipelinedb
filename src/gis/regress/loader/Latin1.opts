# The DBF contains a string with Norweigan characters.
-W LATIN1
