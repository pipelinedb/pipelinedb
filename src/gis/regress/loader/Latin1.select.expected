1|Tårneby in Våler I Solør kommune
